fedimint-cli/src/lib.rs

```diff
@@ -430,6 +431,12 @@ Examples:
     /// Wait for the fed to reach a consensus block count
     WaitBlockCount { count: u64 },
 
+    /// Just start the `Client` and wait
+    Wait {
+        /// Limit the wait time
+        seconds: Option<f32>,
+    },
+
```

Comment:
Great command! I always generated invoices and waited on them as a workaround :see_no_evil: